dist/datasource.js

```diff
@@ -141,6 +141,19 @@ function (angular, _, dateMath, moment) {
       var dataSource = this;
       var from = dateToMoment(options.range.from, false);
       var to = dateToMoment(options.range.to, true);
+      var timeZone = dataSource.periodGranularity;
+
+      if(dataSource.periodGranularity=="dashboard") {
+        if(options.timezone!="") {
+            timeZone = options.timezone
+         }
+        if(options.timezone=="browser") {
+            timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone
+         }
+       }
```

Comment:
```suggestion
        if(options.timezone=="browser") {
            timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone
         }
else if(options.timezone!="") {
            timeZone = options.timezone
         }
       }
```

Reply:
I think there should also be else clause that says console log error "dashbaord" not sending timezone. are you on grafana 7., ?